src/utils/pgwire/src/pg_message.rs

```diff
@@ -197,14 +197,19 @@ impl FeBindMessage {
             .collect();
         // Read ResultFormatCode
         let len = buf.get_i16();
+        let format_codes = (0..len).map(|_| buf.get_i16()).collect::<Vec<_>>();
+        let all_elements_are_equal = format_codes.iter().all(|&x| x == format_codes[0]);
 
-        assert!(len==0||len==1,"Only support default result format(len==0) or uniform result format(len==1), can't support mix format now.");
+        assert!(
+            all_elements_are_equal,
+            "Can't support mix result format now."
+        );
```

Comment:
Should we return err here?

Reply:
Why there are format code beyond 0 and 1?

Reply:
BTW, seems like previously we use length for format code, which is not correct?

Reply:
Great, let's add these into comments

Reply:
If the format code is passed from client, we should return error rather panic, otherwise we may be attacked.